lib/super_digit.rb

```diff
@@ -1,15 +1,34 @@
 # Superdigit
 
-# Time Complexity - ?
-# Space Complexity - ?
+# Time Complexity - O(n) where n is the amount of times the sum is greater than 10 
+# Space Complexity - O(n) where n is the number of digits in n
 def super_digit(n)
```

Comment:
Notice that the while loop is O(n), but you're then calling superdigit on the sum.